daemon/changes.go

```diff
@@ -17,21 +18,17 @@ func (daemon *Daemon) ContainerChanges(job *engine.Job) error {
 		return err
 	}
 
-	outs := engine.NewTable("", 0)
 	changes, err := container.Changes()
 	if err != nil {
 		return err
 	}
 
-	for _, change := range changes {
-		out := &engine.Env{}
-		if err := out.Import(change); err != nil {
-			return err
-		}
-		outs.Add(out)
+	data, err := json.Marshal(changes)
```

Comment:
I think here is better to use `json.NewEncoder` to encode changes right to `job.Stdout`